appveyor.yml

```diff
@@ -8,7 +8,7 @@ environment:
   NOSE_FILTER: not unix and not fixme and not winfixme
   PYWIN_PATH: C:\projects\integrations-core\.cache\pywin32-py2.7.exe
   SKIP_LINT: true
-  DD_AGENT_BRANCH: master
+  DD_AGENT_BRANCH: jaime/pdhwheel
```

Comment:
Must be switched back to `master` after https://github.com/DataDog/dd-agent/pull/3691 is merged.